library/general/requiredprobes/template.yaml

```diff
@@ -29,32 +36,47 @@ spec:
     - target: admission.k8s.gatekeeper.sh
       rego: |
         package k8srequiredprobes
-
+        import data.lib.exempt_container.is_exempt
```

Comment:
Update `src/general/requiredprobes/src.rego` with the rego changes you are trying to make